packages/command-palette/src/index.tsx

```diff
@@ -215,6 +215,16 @@ export interface CommandPaletteProps {
 	userCapabilities: { [ key: number ]: { [ key: string ]: boolean } };
 }
 
+const COMMAND_PALETTE_MODAL_OPEN_CLASSNAME = 'command-palette-modal-open';
+// We need to change the `overflow` of the html element because it's set to `scroll` on _reset.scss
+// Ideally, this would be handled by the `@wordpress/components` `Modal` component,
+// but it doesn't have a `htmlOpenClassName` prop to go alongside `bodyOpenClassName`.
+// So we need to toggle both classes manually here.
+const toggleModalOpenClassnameOnDocumentHtmlElement = ( isModalOpen: boolean ) => {
+	document.documentElement.classList.toggle( COMMAND_PALETTE_MODAL_OPEN_CLASSNAME, isModalOpen );
+	document.body.classList.toggle( COMMAND_PALETTE_MODAL_OPEN_CLASSNAME, isModalOpen );
+};
```

Comment:
Since we're solving a Calypso specific issue (i.e. WP Admin does not need to change the html/body classes), wouldn't it be a better idea to define this behavior in Calypso?

An idea would be to re-use the `onClose` prop and add a new `onOpen` one:

```
<AsyncLoad
		require="@automattic/command-palette"
		placeholder={ null }
		// ...
		onClose={ () => toggleModalOpenClassnameOnDocumentHtmlElement( false ) }
		onOpen={ () => toggleModalOpenClassnameOnDocumentHtmlElement( true ) }
	/>
```

Reply:
I guess we can leave this as-is too, since the class we're adding is generic enough that could be used for other scenarios in WP Admin